ssh_config.5

```diff
@@ -683,6 +683,45 @@ Valid options are:
 and
 .Cm sha256
 (the default).
+.It Cm ForkAfterAuthentication
```

Comment:
Should these new options also be listed in [ssh.1](https://github.com/openssh/openssh-portable/blob/master/ssh.1#L469-L562) under `-o option`?

Reply:
Indeed! I missed that place in the manpages. Fixed.